Makefile

```diff
@@ -6,6 +6,9 @@ CHECKUP_IMAGE_TAG ?= devel
 GO_IMAGE_NAME := docker.io/library/golang
 GO_IMAGE_TAG := 1.19.4-alpine3.17
 
+LINTER_IMAGE_NAME := docker.io/golangci/golangci-lint
+LINTER_IMAGE_TAG := v1.50.1
+
```

Comment:
Shouldn't it be part of the builder image?

Reply:
There is no `builder` image per se.
The official go image is being used to unit-test and build the checkup's binary.
The official `golangci-lint` image is being used to lint the image.

This approach saves us the need to maintain a `builder` image with all the tooling.